api/base/filters.py

```diff
@@ -296,7 +299,10 @@ def query_params_to_odm_query(self, query_params):
             query_parts = []
             for field_name, params in filters.iteritems():
                 for group in params:
-                    query = Q(field_name, group['op'], group['value'])
+                    value = group['value']
+                    if value == 'null':
```

Comment:
Converting input values from the client to their corresponding Python values is the responsibility of `convert_value`. Please move this.